tests/foundry/test/PlainPrankTest.t.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.13;
+
+import "forge-std/Test.sol";
+import "src/Prank.sol";
+
+contract AdditionalToken {
+    address public immutable owner;
+    uint256 public count;
+
+    constructor() {
+        owner = msg.sender;
+        count = 0;
+    }
+
+    function incrementCount() public {
+        require(msg.sender != owner);
+        count = count + 1;
+    }
+}
+
+contract PlainPrankTest is Test {
+
+    function test_startPrank_true() public {
+        AdditionalToken token = new AdditionalToken();
+        vm.startPrank(address(token));
+        token.incrementCount();
+        vm.stopPrank();
+        assert(token.count() == 1);
+    }
+
+    function test_startPrankWithOrigin_true() public {
+        AdditionalToken token = new AdditionalToken();
+        vm.startPrank(address(token), address(token));
+        token.incrementCount();
+        vm.stopPrank();
+        assert(token.count() == 1);
+    }
+
+    function test_stopPrank_true() public {
+        vm.stopPrank();
+        vm.stopPrank();
+    }
+
+    function test_stopPrank_true2() public {
+        AdditionalToken token = new AdditionalToken();
+        vm.startPrank(address(token));
+        token.incrementCount();
+        assert(token.count() == 1);
+    }
+
+    function testFail_startPrank_false() public {
```

Comment:
I don't understand what is tested here and in which line the test case is expected to fail.

Reply:
My understanding is that the test will fail the second time `startPrank` is called since `stopPrank` has not been called before. But I don't know if this is the sole purpose of this test. In any case, I'll isolate this behavior into a new one.